tools/app-provisioning-tool/app-provisioning-lib/app-provisioning-lib.csproj

```diff
@@ -40,8 +40,4 @@
     </EmbeddedResource>
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != 'NET8_0_OR_GREATER'">
```

Comment:
Not needed, this is a net9 app

Reply:
we had issues w/the config generator on .net 6, so I'm not sure we want to take these changes. @jmprieur thoughts?

Reply:
Yes, these ones are right
the wrong one was comparing the TargetFrameworkIdentifier to netcoreapp